src/main/java/digital/slovensko/autogram/server/dto/ServerSigningParameters.java

```diff
@@ -164,7 +164,7 @@ public void validate(MimeType mimeType) {
         }
 
         if (level.getSignatureForm() == SignatureForm.XAdES) {
-            if (!isXMLMimeType(mimeType) && !isXDCMimeType(mimeType) && container == null)
+            if (!isXMLMimeType(mimeType) && !isXDCMimeType(mimeType) && !isAsiceMimeType(mimeType) && container == null)
```

Comment:
toto treba? Asic je conatiner, takže ak container nie je null, nie je to ani asic.

Reply:
Toto je kvoli tomu ked posles payloadMimeType asic